app/frontend/courses/review/components/CoursesReview__Editor.res

```diff
@@ -559,7 +559,7 @@ let gradePillHeader = (evaluationCriteriaName, selectedGrade, gradeLabels) =>
 
 let gradePillClasses = (selectedGrade, currentGrade, passgrade, send) => {
   let defaultClasses =
-    "course-review-editor__grade-pill border-gray-300 py-1 px-2 text-sm flex-1 font-semibold transition " ++
+    "course-review-editor__grade-pill border-gray-300 flex items-center justify-center py-1 ps-1 pe-1 text-sm flex-1 font-semibold transition " ++
```

Comment:
We don't need to use logical properties if we are adding the same amount of padding on both sides. The `px-*` will work fine in this case